test/e2e/lunchApp.js

```diff
@@ -0,0 +1,29 @@
+var conf = require('../../nightwatch.conf.js');
+const email = 'gbartrum2f@devbridge.com';
+const password = 'AyBrDdHAsc';
+
+module.exports = {
+  before: function (browser) {
+    browser.url(browser.launchUrl);
+  },
+  'Login as admin': function (browser) {
+    let login = browser.page.login();
+    login
+      .waitForPage()
+      .enterCredentials(email, password)
+      .assertCredentials(email, password);
+  },
+  'Make an order': function (browser) {
+    let mainDashboard = browser.page.mainDashboard();
+    let ordersHistory = browser.page.ordersHistory();
+    mainDashboard.waitForPage();
+    mainDashboard.orderFirstItem();
+    ordersHistory.navigate();
+    ordersHistory.waitForPage();
+    ordersHistory.searchOrder(email);
+    ordersHistory.assertItemFound(email);
```

Comment:
This is probably only asserting that person have an order? If you would run test case for few times this would be true even if you skip adding dish. Better solution would be to navigate to order history, search for user and confirm it is not there. Order lunch. navigate to order history. search for user and confirm it is there. This would work and be reliable test. After the test "tear-down" would  be needed to remove ordered lunch, but it could be a test case too